build/scripts/build-client-docker

```diff
@@ -20,18 +20,23 @@ fi
 
 CLIENT_IMAGE_VERSION=`echo $DOCKER_CLIENT_TAG | cut -d ':' -f2`
 
-docker build --build-arg BUILD_DATE="`date --rfc-3339=seconds`" \
-             --build-arg VERSION="$CLIENT_IMAGE_VERSION" \
-             --squash \
-             --no-cache \
-             -f ${DOCKER_FILE_LOCATION}/${docker_file} \
-             -t ${DOCKER_CLIENT_TAG} \
-             ${DOCKER_FILE_LOCATION}
+docker buildx create --use --name multiarch-builder
+docker buildx inspect --bootstrap
+
+docker buildx build --build-arg BUILD_DATE="`date --rfc-3339=seconds`" \
+                    --build-arg VERSION="$CLIENT_IMAGE_VERSION" \
+                    --platform linux/amd64,linux/arm64 \
+                    --squash \
+                    --no-cache \
+                    --push \
```

Comment:
Same here - pushing the image takes place on line 39.

Reply:
@talhabinrizwan PTAL ^

Reply:
docker save isn't really pushing, is it?

Reply:
> docker save isn't really pushing, is it?

No, it does not, but `docker push` was somehow removed from that block of code.

<img width="640" alt="Screenshot 2024-06-12 at 12 42 12" src="https://github.com/percona/pmm/assets/81549/32ad1764-bb87-46f4-833b-0f58d332971a">

Reply:
ah, right, you mentioned line 39, that's why I didn't look above